docker/deb-ci/Dockerfile_template

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:CYCLUS_UBUNTU_MAJOR_VERSION.04
+
+# Because we used specific Ubuntu version can't use cyclus-deps docker container
+RUN apt-get -y --force-yes update
+
+RUN apt-get install -y --force-yes \
+    cmake \
+    make \
+    libboost-all-dev \
+    libxml2-dev \
+    libxml++2.6-dev \
+    libsqlite3-dev \
+    libhdf5-serial-dev \
+    libbz2-dev \
+    coinor-libcbc-dev \
+    coinor-libcoinutils-dev \
+    coinor-libosi-dev \
+    coinor-libclp-dev \
+    coinor-libcgl-dev \
+    libblas-dev \
+    liblapack-dev \
+    g++ \
+    libgoogle-perftools-dev \
+    git \
+    python \
+    python-tables \
+    python-numpy \
+    python-nose
+
+COPY . /cyclus 
+WORKDIR cyclus
+RUN mkdir -p deb
+WORKDIR /cyclus/deb
+RUN cmake -DCPACK_PACKAGING_INSTALL_PREFIX=/usr/local -DCMAKE_INSTALL_PREFIX=/usr/local -DCPACK_HASH_NAME=CYCLUS_REPO_HASH -DCPACK_SYSTEM_NAME=ubuntu.CYCLUS_UBUNTU_MAJOR_VERSION.04 ..
```

Comment:
There is a discussion about it: 
http://unix.stackexchange.com/questions/30/where-should-i-put-software-i-compile-myself

I think people agree that `/usr` is for the distribution stuff, `/usr/local` for the safe, and `/opt` for you own compiled tool.
According to that, we should `cyclus` in `/opt`...
But /opt is not in the $PATH by default on `ubuntu`, so I think I prefer to keep the installation in `/usr/local` to simplify users installation...